pkg/discovery/worker/group_discovery_worker.go

```diff
@@ -155,6 +157,29 @@ func (worker *k8sEntityGroupDiscoveryWorker) buildSidecarContainerSpecGroup(side
 	return groupsDTOs
 }
 
+func (worker *k8sEntityGroupDiscoveryWorker) buildUnknownNodesGroup(unknownNodes []string) []*proto.GroupDTO {
+	var groupsDTOs []*proto.GroupDTO
+	if len(unknownNodes) <= 0 {
+		return groupsDTOs
+	}
+	id := fmt.Sprintf("Unknown-Nodes-%s", worker.targetId)
+	displayName := "Unknown Nodes"
```

Comment:
s/"Unknown Nodes"/"Unknown State Nodes"/

Reply:
Updated display name to `All Nodes (State Unknown)`